docs/_includes/permalinks.html

```diff
@@ -5,8 +5,11 @@
 $.each(headers, function(i, header){
     var color = colors[i];
     $(header).each(function () {
-        var href=$(this).attr("id");
-        $(this).append('<a class="headerlink" style="color:' + color + '" href="#' + href + '" title="Permanent link">¶</a>')
+    	$header = $(this);
+    	if(!$header.hasClass("noTOC")){
```

Comment:
Nice - "noTOC". We may need documentation for the documentation. :)

It may make sense to add a small note about this new feature in here - https://docs.google.com/document/d/17guuaxb0xsfutBCzyj2CT6OZiFnMu9w4PzoILXhRXSo/edit

Reply:
Started a new section for the theme and added info on the permalinks, TOC, and noTOC override!